rust/osm2lanes/Cargo.toml

```diff
@@ -9,3 +9,7 @@ authors = ["Dustin Carlino <dabreegster@gmail.com>"]
 log = "0.4"
 serde = { version = "1", features = ["derive"] }
 serde_json = "1"
+reqwest = { version = "0.11", features = ["blocking", "json"] }
+
+[features]
+overpass = []
```

Comment:
connect these? but how?

Reply:
Ah, you can do something like

```
request = { version = "0.11", optional = true }

[features]
overpass = ["reqwest", "reqwest/blocking", "reqwest/json"]
```

Just guessing that's what you're looking for?

Reply:
yup thanks